charts/astronomer/templates/registry/registry-statefulset.yaml

```diff
@@ -53,6 +53,7 @@ spec:
 {{- end }}
     spec:
       serviceAccountName: {{ template "registry.serviceAccountName" . }}
+      securityContext: {{ toYaml .Values.registry.podSecurityContext| nindent 8 }}
```

Comment:
@Shubham-astro why are you removing the securityContext inside the condition and leaving the condition removed and also if you now adding them without condition we have to handles for openshift as well